utils/subreddit.py

```diff
@@ -39,6 +39,9 @@ def get_subreddit_undone(submissions: list, subreddit, times_checked=0):
                 f'This post has under the specified minimum of comments ({settings.config["reddit"]["thread"]["min_comments"]}). Skipping...'
             )
             continue
+        if  settings.config['settings']['storymode'] :
+                if  not submission.is_self   :
+                    continue
```

Comment:
Where is this if statement for and why is it only for story mode?

Reply:
It will check if story mode is enabled and check for the post for is_self if not it will check next post. It can also called strict story mode i had some version with non strict story mode codes